free-3.md

```diff
@@ -45,6 +67,7 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There aren't any other DSLs in this domain.  I know because I spend a decent amount of time on smash-specific subreddits and if a DSL existed in this domain I would've heard about it.  A DSL in this domain likely doesn't exist because there isn't a huge need for it.  Theory-crafting can be done without a DSL by manually going over frame data.
```

Comment:
True, it does seem like there is a way to do this without a dsl, and I'm not sure how easy it is to do, but it certainly seems WAY easier to do in a DSL than by looking at frame data and/or setting up a TAS.